tests/security/secure_boot.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test the signed/unsigned module and kernel if can be loaded when secure-boot is enabled
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
```

Comment:
Any test case number?

Reply:
no test case number, there is no poo for it.

Reply:
> no test case number, there is no poo for it.

Please open a poo then.